Dfe.Data.SearchPrototype/Web/Tests/Unit/Models/Factories/SearchResultsFactoryTests.cs

```diff
@@ -0,0 +1,80 @@
+﻿using Dfe.Data.SearchPrototype.Common.Mappers;
+using Dfe.Data.SearchPrototype.SearchForEstablishments.Models;
+using Dfe.Data.SearchPrototype.Web.Mappers;
+using Dfe.Data.SearchPrototype.Web.Models.Factories;
+using Dfe.Data.SearchPrototype.Web.Tests.Shared.TestDoubles;
+using Dfe.Data.SearchPrototype.Web.Tests.Unit.TestDoubles;
+using Moq;
+using Xunit;
+
+namespace Dfe.Data.SearchPrototype.Web.Tests.Unit.Models.Factories
+{
+    public sealed class SearchResultsFactoryTests
+    {
+        [Fact]
+        public void CreateViewModel_ValidInput_CallsMappers()
+        {
+            // arrange
+            Mock<IMapper<EstablishmentResults?, List<Web.Models.ViewModels.Establishment>?>> mockEstablishmentResultsToEstablishmentsViewModelMapper =
+                EstablishmentResultsToEstablishmentsViewModelMapperTestDouble.MockFor([]);
+
+            Mock<IMapper<FacetsAndSelectedFacets, List<Web.Models.ViewModels.Facet>?>> mockEstablishmentFacetsToFacetsViewModelMapper =
+                FacetsAndSelectedFacetsToFacetsViewModelMapperTestDouble.MockFor([]);
+
+            //act
+            SearchResultsFactory searchResultsFactory =
+                new(mockEstablishmentResultsToEstablishmentsViewModelMapper.Object,
+                    mockEstablishmentFacetsToFacetsViewModelMapper.Object);
+
+            // assert/verify
+            searchResultsFactory.CreateViewModel(EstablishmentResultsTestDouble.Create(), FacetsAndSelectedFacetsTestDouble.Create());
+
+            mockEstablishmentResultsToEstablishmentsViewModelMapper.Verify(mapper => mapper.MapFrom(It.IsAny<EstablishmentResults?>()), Times.Once());
+            mockEstablishmentFacetsToFacetsViewModelMapper.Verify(mapper => mapper.MapFrom(It.IsAny<FacetsAndSelectedFacets>()), Times.Once());
```

Comment:
I still don't like the way we're writing these tests - I think we should be testing that the mocks are called with the expected values rather than using It.IsAny<>
These test could maybe all be one parameterised test, as they're all testing that whatever is passed into CreateViewModel, the mappers are called

Reply:
I think 
I'll take the hint and make the change rather than use It.IsAny

Reply:
Made the change.